checker/build.xml

```diff
@@ -521,13 +540,15 @@
       <!-- Copied from -run-tests target -->
       <mkdir dir="${build.reports}"/>
 
-      <junit fork="${run.tests.should.fork}"
+        <junit fork="${run.tests.should.fork}"
```

Comment:
the spacing on this target is actually off compared to the others in the doc.  I will make it consistent.